contrib/linux/dosbox-staging.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=dosbox-staging
+GenericName=DOS emulator
+Comment=A modernized DOSBox
+Exec=dosbox
+Icon=dosbox-staging
+Type=Application
+Keywords=DOS,gaming,games,game,emulator
+Categories=Game
```

Comment:
Add few more, please:

    Categories=Game;System;Emulator;

Reply:
If I add more categories, I get the following issue when validating:
```
[wouter@wouter-arch linux]$ desktop-file-validate dosbox-staging.desktop dosbox-staging.desktop 
dosbox-staging.desktop: hint: value "Game;System;Emulator;" for key "Categories" in group "Desktop Entry" contains more than one main category; application might appear more than once in the application menu
```

Reply:
What's your version of `desktop-file-utils` (or whatever package provides `desktop-file-validate` on Arch).

I tested few different entries using desktop-file-utils-0.24-1 and none showed this warning (almost all packages in Fedora provide several values in there)…

Anyway - it's ok, we can keep single category (not sure if it should be Emulator or Game) - we probably should ask some Mint or MATE user for opinion here.

Reply:
This is version 0.24:
```
[wouter@wouter-arch Sources]$ pacman -Qs desktop-file-utils 
local/desktop-file-utils 0.24-2
    Command line utilities for working with desktop entries
```

Reply:
I've added the Emulator category